### PR TITLE
V3.10

### PR DESCRIPTION
**Pull request by shwetha-h-p**
Adding roles, playbook to scale nodes on azure_ocp_cluster

### CODE REVIEW

File: templates/hosts_ocp_scale_post.j2 [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+# Gluster Default SC
+openshift_storage_glusterfs_storageclass_default=true
+openshift_storageclass_default=False
+
+{% else %}
+{% endif %}
+
+ansible_ssh_user="{{ admin_user }}"
+ansible_become=true
+openshift_master_api_port=443
```
Comment (marklabonte) [inline templates/hosts_ocp_scale_post.j2]: we should use {{ api_port }} as defined in vars.yml

File: templates/hosts_ocp_scale_post.j2 [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+openshift_master_console_port=443
```
Comment (marklabonte) [inline templates/hosts_ocp_scale_post.j2]: we should use {{ api_port }} as defined in vars.yml

Comment (hornjason) [reply templates/hosts_ocp_scale_post.j2]: Agreed, please use the defined variable {{ api_port }}

File: templates/hosts_ocp_scale_pre.j2 [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+# Gluster Default SC
+openshift_storage_glusterfs_storageclass_default=true
+openshift_storageclass_default=False
+
+{% else %}
+{% endif %}
+
+ansible_ssh_user="{{ admin_user }}"
+ansible_become=true
+openshift_master_api_port=443
```
Comment (marklabonte) [inline templates/hosts_ocp_scale_pre.j2]: we should use {{ api_port }} as defined in vars.yml

File: templates/hosts_ocp_scale_pre.j2 [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+openshift_master_console_port=443
```
Comment (marklabonte) [inline templates/hosts_ocp_scale_pre.j2]: we should use {{ api_port }} as defined in vars.yml

File: templates/hosts_ocp_scale_post.j2 (hunk 1 above)
Comment (hornjason) [reply templates/hosts_ocp_scale_post.j2]: Updated

**Pull request merged**